catalog/MDCCatalog/MDCNodeListViewController.swift

```diff
@@ -451,22 +468,40 @@ extension MDCNodeListViewController {
     sectionView.addSubview(descriptionLabel)
 
     // sectionTitleLabel to SectionView
-    NSLayoutConstraint(
-      item: sectionView,
-      attribute: .leading,
-      relatedBy: .equal,
-      toItem: sectionTitleLabel,
-      attribute: .leading,
-      multiplier: 1.0,
-      constant: -padding).isActive = true
-    NSLayoutConstraint(
-      item: sectionView,
-      attribute: .trailing,
-      relatedBy: .equal,
-      toItem: sectionTitleLabel,
-      attribute: .trailing,
-      multiplier: 1.0,
-      constant: padding).isActive = true
+    let preiOS11Behavior = {
+      NSLayoutConstraint(
+        item: sectionView,
+        attribute: .leading,
+        relatedBy: .equal,
+        toItem: sectionTitleLabel,
+        attribute: .leading,
+        multiplier: 1.0,
+        constant: -self.padding).isActive = true
```

Comment:
For Swift, assignment to `isActive` is expected?

Reply:
I'm not sure, but this is a pattern I've seen elsewhere.